README.md

```diff
@@ -19,7 +19,7 @@ See `test/Test.agda` for an example.
 - [x] Compile if/then/else [#13](https://github.com/agda/agda2hs/pull/13)
 - [ ] Literals in patterns
 - [x] Use some Haskell syntax ADT and a proper pretty printing library [#4](https://github.com/agda/agda2hs/pull/4)
-- [ ] Map instance arguments to Haskell type classes (definitions and use) [#3](https://github.com/agda/agda2hs/pull/3)
+- [ ] Map instance arguments to Haskell type classes (definitions and use) [#3](https://github.com/agda/agda2hs/pull/3),  [#31](https://github.com/agda/agda2hs/pull/31),  [#37](https://github.com/agda/agda2hs/pull/37)
```

Comment:
Maybe it's time to retire this list and use issues instead...

Reply:
I agree we should not continue using it as a TODO thing, but it's nice to keep a list of features we currently support in the README (so that one does not need to go through the test files).

Reply:
I removed my changes for now.